NineChronicles.Headless/BlockChainService.cs

```diff
@@ -83,8 +83,14 @@ public UnaryResult<bool> PutTransaction(byte[] txBytes)
                 {
                     Log.Debug("PutTransaction: (nonce: {nonce}, id: {id})", tx.Nonce, tx.Id);
                     Log.Debug("StagedTransactions: {txIds}", string.Join(", ", _blockChain.GetStagedTransactionIds()));
-                    _blockChain.StageTransaction(tx);
-                    _swarm.BroadcastTxs(new[] { tx });
+#pragma warning disable CS8632
+                    Exception? validationExc = _blockChain.Policy.ValidateNextBlockTx(_blockChain, tx);
```

Comment:
It will be better to leave error logs (e.g., `_logger.Error`) for debugging. It is not required.

Reply:
Add message in https://github.com/planetarium/NineChronicles.Headless/pull/2103/commits/38dc13f43684ed12581fac466ef803c9dfa4fa9d